### DISCUSSION TITLE
Add a block, manual input of a variable or value, to execute the next block

### DISCUSSION THREAD

**Discussion by austimchik** [postmanlabs/postman-flows#172] [Ideas]
There are cases when to execute the next request, it needs to pass a value, for example, a number from an SMS. Without it, this request will not work and it turns out it will always fall. I see it like this, a block with manual values that is waiting for manual input to execute the next query.

**Comment by dom-techwriter** [comment]
I was just about to submit this idea as well! This would be a great little feature for our clients to be able to test out flows and see what their end user will need to do as well (like the example @austimchik gave - providing a verification code or something like that).
